index.html

```diff
@@ -4,6 +4,11 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>JavaScript Foundations</title>
+    <script src="exercise-00.js"></script>
+    <script src="exercise-01.js"></script>
+    <script src="exercise-02.js"></script>
+    <script src="exercise-03.js"></script>
+    <script src="exercise-04.js"></script>
   </head>
   <body></body>
```

Comment:
- [ ] Add content to body. Otherwise Netlify will not run any deploy in preview environment.